.travis.yml

```diff
@@ -1,24 +1,26 @@
 language: c
+git:
+  submodules: false
+
+env: B=autotools
+
 install:
-  - sudo apt-get update -qq
-  - sudo apt-get install -qq curl
   - curl http://download.opensuse.org/repositories/home:/laszlo_budai:/syslog-ng/xUbuntu_12.04/Release.key | sudo apt-key add -
```

Comment:
I have also given this second though before. The OS packages should provide
good enough documentation about package requirements.

However, we could try to validate requirements via `strace` as a CI
enhancement if you see value in that.

On Tue, May 24, 2016 at 11:25 AM, Budai Laszlo <notifications@github.com>
wrote:

> In .travis.yml
> <https://github.com/balabit/syslog-ng/pull/1051#discussion_r64356974>:
>
> >  install:
> > -  - sudo apt-get update -qq
> > -  - sudo apt-get install -qq curl
> >    - curl http://download.opensuse.org/repositories/home:/laszlo_budai:/syslog-ng/xUbuntu_12.04/Release.key | sudo apt-key add -
>
> OK, but previously we had to install it and that time Travis was also a
> development CI and not a desktop system ;-)
> ( and travis.yml could document how to setup a build environment to build
> syslog-ng from source even on a desktop system, but I'm ok with this change
> )
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/balabit/syslog-ng/pull/1051/files/ab476486121cb428ac95ec928d9925b34463ed04..74dda55d0f0fca499ea92d6663b84eedfbf848f6#r64356974>
>